plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -44,6 +44,7 @@
 import java.util.Optional;
```

Comment:
Please update commit message to `Make BigQuery views cache ttl configurable`
In general we try to follow https://chris.beams.io/posts/git-commit/ for commit messages

Reply:
Done.